rpm-testing/install.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+
+#
+# Determine if search capabilities are enabled
+#
+
+search_enabled="true"
+if [ "${SEARCH_ENABLED}" = "no" ] || [ "${SEARCH_ENABLED}" = "false" ] || [ "${SEARCH_ENABLED}" = "0" ]; then
+    search_enabled="false"
+fi
+echo "---------------- DEBUG: SEARCH"
+echo "Enabled: ${search_enabled}"
+echo "------------------------------"
+
+
+#
+# Configure repository
+#
+
+build_local_repository="false"
```

Comment:
With the current setup, you need to build all archivematica-extras rpm packages locally in order to install. 

I think we can can add the centos-extra official repo always because testing new built packages will still work. As new packages are built with different (higher) versions than current ones, and yum will automatically install the package with the higher version.

Reply:
We can set the local repository with a higher priority: https://wiki.centos.org/PackageManagement/Yum/Priorities